pages/web-literacy/WebLiteracyPage.jsx

```diff
@@ -17,11 +17,23 @@ var topicContent = weblitcontent.topics;
 var webLitSkillsContent = weblitcontent.webLitSkills;
 var makeLinksFromWebLitSkills = require("./MakeLinksFromWebLitSkills.jsx");
 
+function convertToRoute(str) {
+  str = str || "";
+  return str.toLowerCase().replace(" ", "-");
+}
+
+function parseRoute(route) {
+  route = route || "";
+  return route.split("-").map(function(item) {
+    return item.charAt(0).toUpperCase() + item.substr(1).toLowerCase();
+  }).join(" ");
+}
```

Comment:
I considered making these a module that we can require in, and I'm fine with doing that if we have strong opinions about it.

only the convertToRoute is used in another file, and seemed simple enough to duplicate it, but we can make a module for it if we want.